src/test/java/FormAuthenticationTest.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class FormAuthenticationTest extends BasicSetupTest{
+    @Test
+    public void formAuthenticationTest() throws InterruptedException {
+
+        browser.get("https://the-internet.herokuapp.com/login");
+        WebElement usernameField = browser.findElement(By.cssSelector("#username"));
+        WebElement passwordField = browser.findElement(By.cssSelector("#password"));
+        usernameField.sendKeys("tomsmith");
+        passwordField.sendKeys("SuperSecretPassword!");
+        Thread.sleep(1000);
```

Comment:
No need for sleep here as well, typing happens immediately as the element is already available